solidity/test/data/moving-funds.ts

```diff
@@ -302,87 +372,88 @@ export const MultipleTargetWalletsAndDivisibleAmount = {
  * - 2 P2PKH and 1 P2WPKH outputs that matches the target wallets from
  *   the commitment
  * - The total transacted amount is distributed unevenly over the target wallets
- * - 6+ on-chain confirmations of the sweep transaction
+ * - 6+ on-chain confirmations of the transaction
  * - This is not a valid moving funds transaction that should be rejected
  *   because amounts are not evenly distributed.
  */
-export const MultipleTargetWalletsButAmountDistributedUnevenly = {
-  wallet: {
-    // Uncompressed public key for the pubKeyHash `0x7ac2d9378a1c47e589dfb8095ca95ed2140d2726`:
-    //    04ee067a0273f2e3ba88d23140a24fdb290f27bbcd0f94117a9c65be3911c5c04efc314aa6ecfea6a43232df446014c41fd7446fe9deed7c2b054f7ea36e396306
-    // X: ee067a0273f2e3ba88d23140a24fdb290f27bbcd0f94117a9c65be3911c5c04e
-    // Y: fc314aa6ecfea6a43232df446014c41fd7446fe9deed7c2b054f7ea36e396306
-    // ecdsaWalletID = keccak256(XY)
-    ecdsaWalletID:
-      "0x4ad6b3ccbca81645865d8d0d575797a15528e98ced22f29a6f906d3259569863",
-    pubKeyHash: "0x7ac2d9378a1c47e589dfb8095ca95ed2140d2726",
-    state: walletState.MovingFunds,
-  },
+export const MultipleTargetWalletsButAmountDistributedUnevenly: MovingFundsTestData =
```

Comment:
Should we add `expectedMovedFundsSweepRequests` here and cover it with unit tests? If not, can we explain in the comment why we are not doing it?

Reply:
We shouldn't as this scenario will revert. I'll drop a comment above `expectedMovedFundsSweepRequests` in the `MovingFundsTestData` explaining where this parameter is expected.

Reply:
Can we take the opportunity and improve documentation for `MultipleInputs` and `SingleProvablyUnspendable` by noting they are not valid moving funds commitments?

Reply:
See https://github.com/keep-network/tbtc-v2/pull/244/commits/8ac2951679f941e1f90e0b7efd40c65285b01505